Uchu.Core/UchuServer.cs

```diff
@@ -262,28 +263,31 @@ public async Task SetupApiAsync()
         /// <param name="acceptConsoleCommands">Whether to handle console commands or not</param>
         public async Task StartAsync(Assembly assembly, bool acceptConsoleCommands = false)
         {
-            RegisterDefaultAssemblies(assembly);
-            if (acceptConsoleCommands)
+            using (SentrySdk.Init(Config.SentryDsn))
```

Comment:
You've mentioned the changes work without Sentry. What happens here if Sentry is not configured for the user? Does Sentry just do nothing if `Init` passes in nothing?

Reply:
So by default the `SentryDsn` tag in the `UchuConfiguration` is set to an empty string, for which the sentry docs denote the following:

`        /// <summary>
        /// Initializes the SDK with the specified DSN.
        /// </summary>
        /// <remarks>
        /// An empty string is interpreted as a disabled SDK.
        /// </remarks>`